roles/siege-run/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Coppied from mycourses load-generator.yml example
+- name: Configure Siege
+  template:
+    src: "files/siege.conf"
+    dest: "/.sigerc"
+    seuser: system_u
+
+- name: Push hosts file
+  template:
+    src: "files/hosts.conf"
+    dest: "/usr/local/etc/urls.txt"
+    seuser: system_u
+
+- name: Generating Load
+  command: "siege"
+  register: output
```

Comment:
Just curious, are you registering the output for a particular reason?

Reply:
Nope, its actually configured as silent so that line is pointless.